### PR TITLE
Add support for the new Critical severity level.

### PR DESCRIPTION
**Pull request by jportner**
References

* https://updates.snyk.io/critical-severity-level-195891
* https://snyk.docs.apiary.io/#introduction/overview-and-entities/issue

We should release a new version after merging this. It is not a backwards-incompatible change, so I think a new minor would be fine.

### CODE REVIEW

File: cli/snyk.js [hunk 1]
```diff
@@ -70,10 +70,12 @@ module.exports = class Snyk {
 };
 
 function getSeverities(minimumSeverity) {
-    if (minimumSeverity && minimumSeverity.toLowerCase() === 'high') {
-        return ['high'];
+    if (minimumSeverity && minimumSeverity.toLowerCase() === 'critical') {
+        return ['critical', 'high'];
```
Comment (watson) [inline cli/snyk.js]: I think you meant to only include `critical` in this array right?

```suggestion
        return ['critical'];
```

**Review by watson — COMMENTED** [review]
FYI: I was told by Snyk that the old deprecated API we still use does not support the critical level and that we would have to wait till we moved to the new API before we could get access to it. But I guess this is all that's needed to support it once we do get access to it 🤔

File: cli/snyk.js (hunk 1 above)
Comment (jportner) [reply cli/snyk.js]: Yep, good catch thanks!

**Review by watson — APPROVED** [review]

**Comment by jportner** [comment]
> FYI: I was told by Snyk that the old deprecated API we still use does not support the critical level and that we would have to wait till we moved to the new API before we could get access to it. But I guess this is all that's needed to support it once we do get access to it 🤔

Maybe we shouldn't publish a new version yet, then?
If I have some time I'll try to come up with a test repo with a Critical vulnerability so we can see what happens. Certainly the vulnerability is not just excluded from the API results. Maybe the deprecated API returns it as High severity instead?

**Comment by watson** [comment]
> Maybe we shouldn't publish a new version yet, then?

Yeah I don't see any reason as long as we still use the deprecated API.

> If I have some time I'll try to come up with a test repo with a Critical vulnerability so we can see what happens. Certainly the vulnerability is not just excluded from the API results. Maybe the deprecated API returns it as High severity instead?

All "critical" is just labeled "high" in the deprecated API.

**Pull request merged**